src/puzzle/Board.js

```diff
@@ -898,6 +898,53 @@ pzpr.classmgr.makeCommon({
 				}
 			}
 			return dlist;
+		},
+
+		// Draw borders between opposite-shaded cells.
+		// A cell is shaded if its qsub value is nonzero; unshaded cells are ignored.
+		// A line is drawn between two shaded cells with opposite shades,
+		// or erased between shaded cells with the same shade.
+		// "Cells" outside the grid are treated as having shade 2,
+		// unless there is a cross mark between a shaded cell and the exterior, in which case that cell's shade is used as the exterior shade.
```

Comment:
I'd like this part of the comment to be rewritten. The exterior shade should be treated as the more common case, with the default value of 2 mentioned afterwards. Also, line 908 is currently too long. (we don't have strict lint rules for this, but it currently sticks out compared to the rest)